tests/archives/cvt_archive_test.py

```diff
@@ -136,3 +136,40 @@ def test_add_single_without_overwrite(data, add_mode):
     assert np.isclose(value, low_objective - data.objective)
     assert_archive_elite(data.archive_with_elite, data.solution, data.objective,
                          data.measures, data.centroid, data.metadata)
+
+
+def test_chunked_calculation_short():
+    """Testing accuracy of chunked computation"""
+    archive = CVTArchive(solution_dim=0,
+                         cells=10,
+                         ranges=[(-1, 1), (-1, 1)],
+                         samples=10,
+                         chunk_size=1000,
+                         chunk_distances=False,
+                         seed=1,
+                         use_kd_tree=False)
+    measure_batch = [[-1, -1], [-.75, -.75], [-.5, -.5], [-.25, -.25], [0, 0],
+                     [.25, .25], [.5, .5], [.75, .75], [1, 1]]
+    closest_centroids = archive.index_of(measure_batch)
+    correct_centroids = [9, 9, 9, 8, 1, 5, 5, 5, 2]
+
+    for i, c in enumerate(closest_centroids):
+        assert c == correct_centroids[i]
+
+
+def test_chunked_calculation_long():  # index_of crashes on input of len 1e8
```

Comment:
Is the goal of this test to force an OOM? If so, I think you can remove it -- in general, OOM is system-dependent. Furthermore, forcing an OOM will cause the tests to fail. Our goal with this PR should be just to implement chunking correctly; the OOM test I suggested was more to check that OOM actually happens.

Reply:
This test was there to test inputs that force an OOM. Ok, I'll remove it